openssl/src/ssl/connector.rs

```diff
@@ -91,8 +91,31 @@ impl SslConnector {
             verify_hostname: true,
         })
     }
+
+    pub fn into_ssl_context(self) -> SslContext {
+        self.0
+    }
+
+    pub fn ssl_context_ref(&self) -> &SslContextRef {
+        &*self.0
+    }
+}
+
+impl Deref for SslConnector {
```

Comment:
I'd prefer to not have the deref/derefmut impls here. I want to keep the acceptor/connector a bit more opaque to guide towards the high-level methods.

Reply:
So keep only the into_ssl_context() and into_ssl_context_ref()?

Reply:
Yep